src/App/Jobs/Register_Tamara_WP_Api_Routes_Job.php

```diff
@@ -14,7 +14,7 @@ class Register_Tamara_WP_Api_Routes_Job extends Base_Job {
 
 	public function handle(): void {
 		Route::post( 'tamara/success', [ Main_Controller::class, 'handle_tamara_success' ] )->name( 'tamara-success' );
-		Route::post( 'tamara/cancel', [ Main_Controller::class, 'handle_tamara_cancel' ] )->name( 'tamara-cancel' );
+		Route::get( 'tamara/orders/{wc_order_id}/cancel', [ Main_Controller::class, 'handle_tamara_cancel' ] )->name( 'wp-api-tamara-cancel' );
```

Comment:
@loikimle you can check this commit to see how we use the routes. For General Helper, you should use static method for all and you don't need create an object to that class.

You don't need to use $this->tamara_checkout_wp_plugin, just use `Tamara_Checkout_WP_Plugin::wp_app_instance()`